save-plugins/warn-plugin/src/commonMain/kotlin/org/cqfn/save/plugin/warn/WarnPlugin.kt

```diff
@@ -65,6 +66,17 @@ class WarnPlugin(
         }
     }
 
+    private fun plusLine(
+        warningRegex: Regex,
+        linesFile: List<String>,
+        lineNum: Int): Int {
+        var x = 1
+        while (lineNum - 1 + x < linesFile.size && warningRegex.find(linesFile[lineNum - 1 + x]) != null) {
```

Comment:
What if `x` becomes larger than the number of lines? It means, that the user has written warnings in the end of the file, and there are no lines left to calculate line numbers. I believe in this case we should log a warning, while now you are silently setting the last line number